### DISCUSSION TITLE
Forum pasword reset broken?

### DISCUSSION THREAD

**Discussion by BennehBoy** [stm32duino/Arduino_Core_STM32#2139] [General]
My forum account appears to have had the password changed, I've now carried out 4 resets to different strong passwords but am still unable to log in (tried multiple browsers and cleared cookies/cache).  Is there a general issue?

**Comment by fpistm** [comment]
I've forced reactivation.
Since several month forum is soammed by bot. That's why I have enforced the user registration and require new password each 90 days.

**Comment by BennehBoy** [comment]
Thanks Frederic, working now.

**Reply by fpistm** [reply]
Welcome
